tensorflow/matmul.py

```diff
@@ -64,7 +65,12 @@ def _get_aicoe_tensorflow_build_info():
         with open(build_info_path, 'r') as build_info_file:
             build_info = json.load(build_info_file)
         return build_info
-    except Exception:
+    except FileNotFoundError:
+        # This is not an AICoE build so we include default tensorflow build info
+        if LooseVersion(tf.__version__) >= LooseVersion("2.3.0"):
+            build_info = tf.sysconfig.get_build_info()
```

Comment:
This does not correspond to the "AICoE" build info but is rather an upstream information about the build. It has different schema (we don't have control of it) so it might break in other parts when integrating (e.g. syncing into the knowledge base).

You can create a separate function for this info gathering that can result in a separate entry in the performance indicator result. Not to relay on `distutils`, you can catch `AttributeError`. This way the script will not rely on distutils API and will be self-sustaining:
```python
def _get_tensorflow_build_info():
    try:
        return tf.sysconfig.get_build_info()
    except AttributeError:
        return None
```

We should also distinguish AICoE build info and the upstream build info in the resulting dict.

Reply:
Thanks I have added it as a new key `tensorflow_upstream_buildinfo`

Reply:
Cool! Could you also adjust the `tensorflow_upstream_buildinfo` to `tensorflow_aicoe_buildinfo`? I think we do not use it directly as of now.

You can apply similar patch to the rest of the source code related to TensorFlow.

Thanks! Looks good! :+1: